rust-toolchain.toml

```
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "1.76"
+profile = "default"
```

```suggestion
[toolchain]
channel = "1.76"
profile = "default"
components = ["rust-analyzer"]
```

Otherwise LSP will be broken for people who use rust-analyzer provided by rustup.

---

Added it, thanks for letting me know. I'm assuming that this results in unnecessary downloads for people who use `rust-analyzer` from other sources (I think my VS Code extension provisions it itself?), but that's still in line with the trade-off I'm trying to make in this PR.